### ISSUE TITLE
Immediate crash iOS 14.1.

### ISSUE THREAD

**Issue by maydayVFR** [Clon1998/mobileraker#186]
Just updated to latest version today. Was also crashing yesterday but yesterday was my first usage of Mobileraker so previous successful builds n/a. Running in English on 14.1. Looks like an iOS 14.3 vs ios 16 build issue. So I could be out of luck. 
Can upload full crash log at request. 

```

Termination Description: DYLD, Symbol not found: _objc_claimAutoreleasedReturnValue | Referenced from: /private/var/containers/Bundle/Application/880C0DD4-A5FC-4188-A814-EF042B1601EA/Runner.app/Frameworks/flutter_native_splash.framework/flutter_native_splash (which was built for iOS 16.4) | Expected in: /usr/lib/libobjc.A.dylib

```

**Comment by ripinteer** [comment]
Same issue here

**Comment by DelukaDZ** [comment]
Same here on IOS 15.7.6

**Comment by Clon1998** [comment]
Hey,
I am working on it!

**Comment by Clon1998** [comment]
Related to #183

**Comment by FastivProService** [comment]
Good day. It also does not work, immediately closes. iOS 15.7.6

**Comment by Clon1998** [comment]
Hey,
I am still not sure why theses crashes are happening. However, I changed some dependencies and got the app working on:
- IOS16.4
- IOS15.0
- IOS14.1

I am currently building this version for testflight and will post an invitation link to it as soon as the build is ready.
Please note that this version has Notifications disabled due to another issue I was unable to resolve today.

In order to ensure an easier debugging process for me, please completely remove Mobileraker before installing the Testflight version!

**Testflight Link**: WIP (I Will update as soon as available.)

<img width="1283" alt="image" src="https://github.com/Clon1998/mobileraker/assets/10357775/368d0e42-a5bd-4aeb-bf19-fd55a4fd5b98">

**Comment by Clon1998** [comment]
Here ia the TestFlight invitation:

https://testflight.apple.com/join/kO7PuGUL

**Comment by maydayVFR** [comment]
And we're back! Will test thoroughly if you need but the immediate crash appears fixed! Good work! Thank you!

**Comment by maydayVFR** [comment]
Everything appears to be in working order on my end. Printer is there, responding, stats line up. Can wait for others to chime in to close or whatever you need.

**Comment by Clon1998** [comment]
@maydayVFR thanks for your feedback and I am quite happy that it seems to be fixed. Let's wait for more users in this issue to give feedback just to be sure!

Feel free to test stuff and report any findings.
I just recently added Crashlyrics and discovered a several issues users never reported so far.

**Comment by Clon1998** [comment]
@DelukaDZ , @ripinteer , @FastivProService. Can you guys give me feedback if the testflight version managed to resolve the issue for you too?

**Comment by ripinteer** [comment]
My apologies for delayed answer, I have 2.4.1 version downloaded with testflight and it still crashes(
Am have the latest version? How can I help you?

**Comment by Clon1998** [comment]
> My apologies for delayed answer, I have 2.4.1 version downloaded with testflight and it still crashes( Am I have the latest version? How can I help you?

Please try this invite, the new version should be 2.4.2
https://testflight.apple.com/join/kO7PuGUL

**Comment by ripinteer** [comment]
![3A451BDF-F39D-476C-B58B-F208A03B304C](https://github.com/Clon1998/mobileraker/assets/65899470/24946dab-9074-4013-ba5c-69c5a24d3f16)
![A2337864-0A21-4E07-AE70-C3867B6A4155](https://github.com/Clon1998/mobileraker/assets/65899470/eedd24a6-1522-47de-a80d-3cfdca4cb9dd)
Tried it a couple times, still shows 2.4.1

**Comment by maydayVFR** [comment]
@Clon1998 will do. In the meantime I'll try to break it haha.

**Comment by Clon1998** [comment]
> ![3A451BDF-F39D-476C-B58B-F208A03B304C](https://github.com/Clon1998/mobileraker/assets/65899470/24946dab-9074-4013-ba5c-69c5a24d3f16)
> 
> ![A2337864-0A21-4E07-AE70-C3867B6A4155](https://github.com/Clon1998/mobileraker/assets/65899470/eedd24a6-1522-47de-a80d-3cfdca4cb9dd)
> 
> Tried it a couple times, still shows 2.4.1

You might be in a different test group, i will look into that tomorrow. You should get a notification once 2.4.2 is ready

**Comment by ripinteer** [comment]
Updated, now it works as normal. Thank you
I’ll do a little testing now
